# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xorg/drm-linux26-i386.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

The drm Makefile complains about CONFIG_X86_CMPXCHG missing. However I
do not care and it compiles without it, yuck.

  - Rene Rebe <rene@exactcode.de>

--- xc/extras/drm/linux/Makefile.vanilla	2005-02-20 23:07:13.254984432 +0000
+++ xc/extras/drm/linux/Makefile	2005-02-20 23:07:20.380901128 +0000
@@ -279,12 +279,6 @@
 	($(VERSION).$(PATCHLEVEL).$(SUBLEVEL)))
 endif
 
-ifdef ARCHX86
-ifndef CONFIG_X86_CMPXCHG
-$(error CONFIG_X86_CMPXCHG needs to be enabled in the kernel)
-endif
-endif
-
 # This needs to go before all other include paths.
 CC += -I$(DRMSRCDIR)
 
